fake/src/lib.rs

```diff
@@ -198,12 +198,32 @@ pub trait Fake: Sized {
 }
 impl<T> Fake for T {}
 
+fn unique<U: Dummy<Faker> + PartialEq, R: Rng + ?Sized>(rng: &mut R, count: usize) -> Vec<U> {
+    let mut set = Vec::<U>::new();
```

Comment:
Curious whether this would be acceptable in the global namespace.

I couldn't find a [collection](https://doc.rust-lang.org/std/collections/index.html) that only requires [PartialEq](https://doc.rust-lang.org/std/cmp/trait.PartialEq.html) , so here is an ugly one.

If this was to become `pub` it should be documented as not scalable, and should only be used for small `count`.